config/main.py

```diff
@@ -4191,6 +4191,142 @@ def del_user(db, user):
             click.echo("Restart service snmp failed with error {}".format(e))
             raise click.Abort()
 
+
+#
+# 'bmp' group ('config bmp ...')
+#
+@config.group()
+@clicommon.pass_db
+def bmp(db):
+    """BMP-related configuration"""
+
+
+#
+# 'enable' subgroup ('config bmp enable ...')
+#
+@bmp.group()
+@clicommon.pass_db
+def enable(db):
+    """Enable BMP table dump """
+
+
+#
+# 'bgp-neighbor-table' command ('config bmp enable bgp-neighbor-table')
```

Comment:
>'config bmp enable bgp-neighbor-table [](http://example.com/codeflow?start=32&length=37)

Would below cli format be better for understanding and less code?

config bmp --feature bgp-neighbor-table --state enabled
config bmp --feature bgp-neighbor-table --state disabled

Or

config bmp set bgp-neighbor-table enabled
config bmp set bgp-neighbor-table disabled

Or more straightforward

config bmp set bgp-neighbor-table=enabled bgp-rib-in-table=disabled

Reply:
This will require HLD update https://github.com/sonic-net/SONiC/blob/50cb55989649d5392b48b54bb7ca8263c883e162/doc/bmp/bmp.md#3-cli, previously I just follow bgp cli style, if no blocking issue, could we keep it first?

Reply:
Even though, can we have a common function to do db related operation, and just leave simple call in each cli part?

Reply:
done.